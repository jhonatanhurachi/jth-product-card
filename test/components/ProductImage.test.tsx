import React from 'react';
import renderer from 'react-test-renderer';
import { ProductImage } from '../../src/components/ProductImage';
import { product2 } from '../data/products';

describe('ProductImage', () => {
  test('must show component with custom image', () => {
    const wrapper = renderer.create(<ProductImage img={product2.img} />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
