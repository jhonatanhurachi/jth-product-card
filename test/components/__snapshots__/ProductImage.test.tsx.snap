// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage must show component with custom image 1`] = `
<img
  alt="Product"
  className="productImg undefined"
  src="./coffee-mug.png"
/>
`;
